docs/source/usage/geometry_conventions.rst

```diff
@@ -0,0 +1,6 @@
+Geometry Conventions
+====================
+
+Extrinsic XYZ Euler angles are often used to specify rotations (in radians). For fully 3D worlds, quaternions are used and the order convention is [qw, qx, qy, qz].
```

Comment:
Why the "For fully 3D worlds" specification? Euler angles can also represent full 3D orientation.

Maybe worth rewording to e.g.:

3D orientations can be represented using Euler angles or quaternions. _<your original sentences here>_. For simplicity, if you are representing poses in 2D, you can simply use the yaw angle (Z Euler angle), but we recommend using quaternions for generic 3D pose calculations and for interfacing with ROS.

Also,

* Worth saying that in our terminology, we use roll = X, pitch = Y, yaw = Z as well?
* I'd also leave a reference to the Pose class for more info, using the RST syntax `:doc:`Pose </generated/pyrobosim.utils.pose.Pose>`